src/components/app.tsx

```diff
@@ -378,8 +378,8 @@ export class App extends React.PureComponent<IProps, IState> {
       this.setState(newState as IState);
 
       this.LARA = initializeLara();
-      if (activity) {
-        loadLearnerPluginState(activity, teacherEditionMode).then(() => {
+      if (activity && resourceUrl) {
```

Comment:
We wont load plugin state if we don't have a resourceUrl. This seems fine, but now I am wondering when we won't have a resourceUrl ... 🤔